model/device.go

```diff
@@ -41,9 +41,8 @@ type Device struct {
 func (dev Device) Validate() error {
 	err := validation.ValidateStruct(&dev,
 		validation.Field(&dev.ID, uuidNotEmpty),
-		validation.Field(&dev.DesiredAttributes),
-		validation.Field(&dev.CurrentAttributes),
-		validation.Field(&dev.UpdatedTS, validation.Required),
+		validation.Field(&dev.ConfiguredAttributes),
+		validation.Field(&dev.ReportedAttributes),
```

Comment:
Question; should we restrict the allowable size (length) of the attributes as well?

Reply:
I implemented the validation with a maximum size set to 100 (including unit tests)